integration-tests/tests/src/hooks/authenticate-user-before.ts

```diff
@@ -18,7 +18,7 @@
 import { Credentials } from "realm";
 
 export function authenticateUserBefore(): void {
-  before(async function (this: AppContext & Partial<UserContext>) {
+  before("authenticateUserBefore", async function (this: AppContext & Partial<UserContext>) {
```

Comment:
To prevent typos and make it easier to refactor if having to rename this in the future, perhaps `authenticateUserBefore.name` would be better to use than `"authenticateUserBefore"`? (Same for the other occurrences where the name is passed.)